tests/Integration/inc/Engine/Optimization/DelayJs/Admin/Subscriber/restoreDefaults.php

```diff
@@ -1,33 +1,40 @@
 <?php
 
-namespace WP_Rocket\Tests\Integration\inc\Engine\Optimization\DelayJs\Admin\Subscriber;
+namespace WP_Rocket\Tests\Integration\inc\Engine\Optimization\DelayJS\Admin\Subscriber;
 
 use WP_Rocket\Tests\Integration\CapTrait;
 use WPMedia\PHPUnit\Integration\AjaxTestCase;
 
 /**
  * @covers \WP_Rocket\Engine\Optimization\DelayJS\Admin\Subscriber::restore_defaults
  *
- * @group  DelayJs
+ * @group  DelayJS
  */
-class Test_RestoreDefaults extends AjaxTestCase{
+class Test_RestoreDefaults extends AjaxTestCase {
 
 	protected $path_to_test_data = '/inc/Engine/Optimization/DelayJs/Admin/Subscriber/restoreDefaults.php';
 	protected $action = 'rocket_restore_delay_js_defaults';
 
+	public function setUp() {
+		parent::setUp();
+
+		CapTrait::hasAdminCapBeforeClass();
+		CapTrait::setAdminCap();
+	}
+
+	public function tearDown() {
+		parent::tearDown();
+
+		CapTrait::resetAdminCap();
+	}
+
```

Comment:
@engahmeds3ed We can use the setup and teardown methods to handle the capacity setup, and make sure it's correctly reset between & after tests